packages/react-vanilla-components/__tests__/components/TelephoneInput.test.tsx

```diff
@@ -103,4 +103,37 @@ describe('Telephone-input', () => {
     expect(input).toHaveLength(1);
     expect(input[0]).toHaveAttribute('aria-describedby', `${f.id}__longdescription ${f.id}__shortdescription`)
   });
+
+  test('it should handle maxlength constraint', async () => {
+    const f = {
+      ...field,
+      maxLength: 5,
+      constraintMessages: {
+        minLength: DEFAULT_ERROR_MESSAGE,
+        maxLength: DEFAULT_ERROR_MESSAGE
+    },
+
+    };
+    let { renderResponse, element } = await helper(f);
+    const input = await renderResponse.findByPlaceholderText(f.placeholder);
+    userEvent.type(input, 'aaaaa');
+    expect(element.getState().value).toEqual('aaaaa');
+    expect(renderResponse.queryByText(f.constraintMessages.minLength)).toBeNull();
```

Comment:
Can you add negative scenario for max length as well ? Also, you don't need to set default error message, we have standard default message already set.

Reply:
Added the negative scenario as well